testing/upload_validation.ts

```diff
@@ -543,6 +544,7 @@ const booleanPackFormulaSchema = zodCompleteObject<Omit<BooleanPackFormula<any>,
   schema: zodCompleteObject<BooleanSchema>({
     type: zodDiscriminant(ValueType.Boolean),
     description: z.string().optional(),
+    displayType: z.nativeEnum(BooleanDisplayType),
```

Comment:
nativeEnum is optional by default? surprising

Reply:
Oops it's not, fixing!